lms/djangoapps/onboarding/models.py

```diff
@@ -253,6 +253,13 @@ def admin_info(self):
         """
         return "%s" % self.admin.email if self.admin else "Administrator not assigned yet."
 
+    def get_first_learner(self):
```

Comment:
Let's please make it a `property` and rename to simply `first_learner`. 
So after this change it would be accessible as something like `organization.first_learner`